src/transformers/models/trillsson_efficientnet/modeling_trillsson_efficientnet.py

```diff
@@ -0,0 +1,610 @@
+    original TensorFlow repo. It can be seen here:
+    https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet/mobilenet.py
+    """
+    if min_value is None:
+        min_value = divisor
+    new_filters = max(min_value, int(filters + divisor / 2) // divisor * divisor)
+    # Make sure that round down does not go down by more than 10%.
+    if new_filters < 0.9 * filters:
+        new_filters += divisor
+    return new_filters
+
+
+class Conv2DSamePadding(nn.Conv2d):
+    """2D Convolutions same padding like TensorFlow,"""
+
+    def __init__(self, in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True):
+        super().__init__(in_channels, out_channels, kernel_size, stride, padding, dilation, groups, bias)
+
+    def forward(self, input_features):
+        in_height = int(input_features.shape[-2])
+        in_width = int(input_features.shape[-1])
+        stride_height, stride_width = self.stride
+        kernel_height, kernel_width = self.kernel_size
+        dilation_height, dilation_width = self.dilation
+
+        if in_height % stride_height == 0:
+            pad_along_height = max(kernel_height - stride_height, 0)
+        else:
+            pad_along_height = max(kernel_height - (in_height % stride_height), 0)
+
+        if in_width % stride_width == 0:
+            pad_along_width = max(kernel_width - stride_width, 0)
+        else:
+            pad_along_width = max(kernel_width - (in_width % stride_width), 0)
+
+        pad_left = pad_along_width // 2
+        pad_right = pad_along_width - pad_left
+        pad_top = pad_along_height // 2
+        pad_bottom = pad_along_height - pad_top
+
+        padding = (
+            pad_left * dilation_width,
+            pad_right * dilation_width,
+            pad_top * dilation_height,
+            pad_bottom * dilation_height,
+        )
+        output_features = nn.functional.pad(input_features, padding, "constant", 0.0)
+        return nn.functional.conv2d(
+            output_features, self.weight, self.bias, self.stride, self.padding, self.dilation, self.groups
+        )
+
+
+class TrillssonSqueezeExcitation(nn.Module):
+    def __init__(
+        self, config: TrillssonEfficientNetConfig, in_channels: int, out_channels: int, reduction: int = 4
+    ) -> None:
+        super().__init__()
+        self.config = config
+        self.se_squeeze = nn.AdaptiveAvgPool2d(1)
+        self.se_reduce = Conv2DSamePadding(
+            in_channels=out_channels,
+            out_channels=make_divisible(in_channels // reduction, config.depth_divisible_by, config.min_depth),
+            kernel_size=1,
+        )
+        self.se_expand = Conv2DSamePadding(
+            in_channels=make_divisible(in_channels // reduction, config.depth_divisible_by, config.min_depth),
+            out_channels=out_channels,
+            kernel_size=1,
+        )
```

Comment:
```suggestion
        in_channels = make_divisible(in_channels // reduction, config.depth_divisible_by, config.min_depth)
        self.se_expand = Conv2DSamePadding(in_channels,out_channels=out_channels,kernel_size=1)
```
Let's write these calls with 2 lines, same for above. You can just have a variable for both instead of calling the same function twice